src/test/resources/features/role/RoleCreate.feature

```diff
@@ -0,0 +1,10 @@
+# feature to review behaviour of create role
+Feature: RoleCreate
+  Scenario: Roles - Registrar datos de un nuevo rol
+    Given 'SSI-A' home page is loaded
+    And Clicking on Personnel menu on 'Header' page
+    And Click in sub menu 'Roles' of menu 'personnel'
+    And click in button 'New Role' of role list page
+    And set 'role' data in create form page
```

Comment:
Example you should use:
And fill 'Role' form using Data Driver Test on 'Role List' page